src/algebra/group/pi.lean

```diff
@@ -42,7 +42,8 @@ instance monoid [∀ i, monoid $ f i] : monoid (Π i : I, f i) :=
 by refine_struct { one := (1 : Π i, f i), mul := (*), npow := λ n x i, (x i) ^ n };
 tactic.pi_instance_derive_field
 
-@[simp, to_additive pi.nsmul_apply]
+-- the attributes are intentionally out of order. `smul_apply` proves `nsmul_apply`.
+@[to_additive nsmul_apply, simp]
```

Comment:
Perhaps clearer would be to use `attribute [-simp] nsmul_apply` below, or `attribute [simp] pow_apply`

Reply:
Well this comment appears in the same way elsewhere in the library, so this is the established convention for this sort of thing as far as I can tell.